Documentation/DocuBlocks/Rest/Administration/1_structs.md

```diff
@@ -11,8 +11,8 @@ The timestamp when this task was created
 
 @RESTSTRUCT{type,api_task_struct,string,required,}
 What type of task is this [ `periodic`, `timed`]
- - periodic are tasks that repeat periodically
- - timed are tasks that execute once at a specific time
+  - periodic are tasks that repeat periodically
+  - timed are tasks that execute once at a specific time
```

Comment:
Fixes rendering in Swagger UI:
![image](https://user-images.githubusercontent.com/7819991/69810300-dcf12780-11eb-11ea-8ca6-7afbc3829633.png)

Renders incorrectly in online docs however (but that's a separate issue)

Reply:
hm, Isn't this rather: https://swagger.io/docs/specification/data-models/enums/ ?

Reply:
should probably be cleaned up, but not doing that now as there are more important problems with the DocuBlock content and not yet decided what we will do about Swagger & DocuBlocks overall